gatsby-node.ts

```diff
@@ -338,11 +340,7 @@ export const createPages: GatsbyNode<any, Context>["createPages"] = async ({
         )
         createPage({
           path: `/${lang}/${page}/`,
-          component: path.resolve(
-            page === "wallets"
-              ? `src/pages-conditional/${page}/index.js`
-              : `src/pages-conditional/${page}.js`
-          ),
+          component: path.resolve(`src/pages-conditional/${page}.tsx`),
```

Comment:
Oh nice, good call renaming the file 😄

Reply:
Can't remember why I put it in a directory 😅

Reply:
:) well...I like the idea of having folders and having the following structure
```
/page
  /index.tsx // only jsx
  /queries.ts
  /styles.ts
```
to have cleaner and smaller files. Would like to propose this approach in the new Learn page.